tests/prefetch_related/tests.py

```diff
@@ -256,6 +258,77 @@ def test_m2m_then_reverse_one_to_one_object_ids(self):
         self.assertWhereContains(sql, self.author1.id)
 
 
+class LargePrefetchSetTests(TestCase):
+    TEST_SIZE = 2000
+
+    @classmethod
+    def setUpTestData(cls):
+        # Create a large set of books and authors
+        books = [Book(title='Poems') for i in range(0, cls.TEST_SIZE)]
+        Book.objects.bulk_create(books)
+
+        # Re-fetch the books as .bulk_create() will not populate their .pk attribute
+        # on SQLite
+        cls.books = Book.objects.all()
+
+        authors = [Author(name='Charlotte {}'.format(i), first_book=b) for i, b in enumerate(cls.books)]
+        Author.objects.bulk_create(authors)
+        cls.authors = Author.objects.all()
+
+        cls.reader = Reader.objects.create(name='Amy')
+
+        for author, book in zip(cls.authors, cls.books):
```

Comment:
This is going to run a *lot* of separate queries to the database isn't it? Is there any way to bulk to this, or create less data in child tables we dont' need?